src/chrome/lib/destroy.js

```diff
@@ -1,15 +1,14 @@
 // Script injected into the page to trigger removal of any existing instances
 // of the Hypothesis client.
-(function () {
```

Comment:
I'd guess you probably stop eslint from doing this automatically by passing `document` into the function:

```javascript
(function (document) {
  ...
}(window.document));
```